app/models/state_file_base_intake.rb

```diff
@@ -95,4 +93,13 @@ def disqualifying_eligibility_answer
   def has_disqualifying_eligibility_answer?
     disqualifying_eligibility_answer.present?
   end
+
+  def save_nil_enums_with_unfilled
```

Comment:
Added this so that any enum that has a `unfilled` type in the base intakes will replace a nil save with unfilled. I noticed that this was causing problems with the flow explorer because we would set initial value account type value to mail and then the value_changed?(to: "mail") statement in the before_save would not catch it in-order change the nil to unfilled. Seen this issue pop up for other stories as well, so hopefully this will prevent database issues